src/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixer.php

```diff
@@ -0,0 +1,247 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\DoctrineAnnotation;
+
+use Doctrine\Common\Annotations\DocLexer;
+use PhpCsFixer\AbstractDoctrineAnnotationFixer;
+use PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException;
+use PhpCsFixer\Doctrine\Annotation\Token;
+use PhpCsFixer\Doctrine\Annotation\Tokens;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+
+/**
+ * Fixes spaces around commas and assignment operators in Doctrine annotations.
+ */
+final class DoctrineAnnotationSpacesFixer extends AbstractDoctrineAnnotationFixer
+{
+    /**
+     * @var bool
+     */
+    private $aroundParentheses;
+
+    /**
+     * @var bool
+     */
+    private $aroundCommas;
+
+    /**
+     * @var bool
+     */
+    private $aroundAssignments;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        if (null === $configuration) {
+            $this->aroundParentheses = true;
+            $this->aroundCommas = true;
+            $this->aroundAssignments = true;
+
+            parent::configure(null);
+
+            return;
+        }
+
+        if (!count($configuration)) {
+            parent::configure($configuration);
```

Comment:
this method is calling parent in 3 different places
let us call it once on top of the method.

then, it will not confusing if `ignored_tags` parameter obligatory when `$configuration` is not an array or not, because here, in this method, you convert `$configuration` to `null` if there is empty array, which is not an obvious flow